### ISSUE TITLE
Renaming Stack is synced up, but not down

### ISSUE THREAD

**Issue by stefan-niedermann** [stefan-niedermann/nextcloud-deck#861]
Deck Server: `1.2.4` and `1.2.5`
Deck Android: `master`, `1.15.0`

Does the `ETag` maybe not change?

**Comment by desperateCoder** [comment]
@juliushaertl the etag seems not to be updated at stack/list-rename. Could you please check that? As soon as I change the card via web UI, we get an updated name.

**Comment by desperateCoder** [comment]
Blocked by: https://github.com/nextcloud/deck/issues/2866

**Comment by stefan-niedermann** [comment]
Closing in favor of https://github.com/nextcloud/deck/issues/2866
